CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Optimizely Java X SDK Changelog
 
+## 3.0.0
+
+February 13, 2019
+
+The 3.0 release improves event tracking and supports additional audience targeting functionality.
+
+### New Features:
+* Event tracking:
+    * The Track method now dispatches its conversion event _unconditionally_, without first determining whether the user is targeted by a known experiment that uses the event. This may increase outbound network traffic.
```

Comment:
```suggestion
    * The `track` method now dispatches its conversion event _unconditionally_, without first determining whether the user is targeted by a known experiment that uses the event. This may increase outbound network traffic.
```
Sorry for the late change! But I think this could be more consistent with other in-code documentation